docker/examples/README.md

```diff
@@ -88,7 +96,7 @@ Single Node
     - To run the example:-
     ```
     # Run from root of the repo
-    $ docker compose -f docker/examples/jvm/single-node/plaintext/docker-compose.yml up
+    $ IMAGE=<image-name> docker compose -f docker/examples/docker-compose-files/single-node/plaintext/docker-compose.yml up
```

Comment:
nit: I am fine with using variable for passing image name but the example commands should be executable as it is imo. I think right now the person trying to follow these steps will have to check the top documentation and put it together to make a working command.

Reply:
Updated the example commands with the exact image name.